src/reducers/ContractReducer.js

```diff
@@ -35,6 +35,7 @@ const DEFAULT_FIELDS = new Map({
 	createdAt: '',
 	owner: new Map(),
 	clickSaveCounter: 0,
+	downloadedCompilers: new Map({}),
```

Comment:
`Map` is inited in reducer, but `List` is used.
Use `Set` and do not convert it to list in `changeContractCompiler`.